stdlib/runtime/utf8.gr

```diff
@@ -15,6 +15,19 @@ from "runtime/unsafe/wasmi32" include WasmI32
  */
 provide exception MalformedUnicode
 
+/**
+ * Determines if the given byte is a leading byte in a UTF-8 sequence.
+ *
+ * @param byte: The byte to check
+ *
+ * @returns `true` if the byte is a leading byte, `false` otherwise
+ */
+@unsafe
+provide let isLeadingByte = byte => {
```

Comment:
Was this intended to be used somewhere?

Reply:
This is used in `string.gr` so its just a tiny bit of cleanup I noticed while looking for constants and decided to address in this pr.

Reply:
Was this already its own function or did you separate out this logic into a function?

Reply:
> Was this already its own function or did you separate out this logic into a function?

I separated it out, happy to move it to a separate pr but I didn't feel like it was worth it, especially as its just work on generally cleaning up more of our `utf8` logic.